examples/app.js

```diff
@@ -37,14 +37,36 @@
       }
     };
 
-    $timeout(function () {
```

Comment:
Should remove `$timeout` from the injected dependencies above: https://github.com/jtblin/angular-chart.js/pull/327/files#diff-416740f4b69f48495e4894828b2daea6L22